src/Desktop/ColorContrastAnalyzer/ColorContrastConfigBuilder.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Axe.Windows.Desktop.ColorContrastAnalyzer
+{
+    public class ColorContrastConfigBuilder
+    {
+        private int? _maxTextThickness;
+        private int? _minNumberOfColorTransitions;
+        private int? _minSpaceBetweenSamples;
+        private int? _transitionCountDominanceFactor;
+        private AnalyzerVersion? _analyzerVersion;
+        private double? _highConfidenceThreshold;
+        private double? _midConfidenceThreshold;
+        private double? _lowConfidenceThreshold;
+
+        public IColorContrastConfig Build() => new ColorContrastConfig(
+            _maxTextThickness, _minNumberOfColorTransitions, _minSpaceBetweenSamples,
+            _transitionCountDominanceFactor, _analyzerVersion,
+            _highConfidenceThreshold,
+            _midConfidenceThreshold,
+            _lowConfidenceThreshold);
+
+        public ColorContrastConfigBuilder WithMaxTextThickness(int maxTextThickness)
+        {
+            _maxTextThickness = maxTextThickness;
+            return this;
+        }
+
+        public ColorContrastConfigBuilder WithMinNumberColorTransitions(int minNumberOfColorTransitions)
+        {
+            _minNumberOfColorTransitions = minNumberOfColorTransitions;
+            return this;
+        }
+
+        public ColorContrastConfigBuilder WithMinSpaceBetweenSamples(int minSpaceBetweenSamples)
+        {
+            _minSpaceBetweenSamples = minSpaceBetweenSamples;
+            return this;
+        }
+
+        public ColorContrastConfigBuilder WithTransitionCountDominanceFactor(int transitionCountDominanceFactor)
+        {
+            _transitionCountDominanceFactor = transitionCountDominanceFactor;
+            return this;
+        }
+
+        public ColorContrastConfigBuilder WithAnalyzerVersion(AnalyzerVersion analyzerVersion)
+        {
+            _analyzerVersion = analyzerVersion;
+            return this;
+        }
+
+        /// <summary>
+        /// Sets the maximum threshold for high confidence. Default value is 0.1.
```

Comment:
Default value comments in this file need to be updated after latest revision

Reply:
Fixed